lib/connectors/memory.js

```diff
@@ -723,6 +725,62 @@ Memory.prototype.updateAttributes = function updateAttributes(model, id, data, o
   }
 };
 
+Memory.prototype.replace = function replace(model, id, data, options, cb) {
```

Comment:
I find `replace` too vague - is it replacing one or multiple instances? Can we rename this method to `Memory.prototype.replaceById` to match the DAO method exposed on models? Or perhaps `Memory.prototype.replaceInstance`.